doc/PLUGINS.md

```diff
@@ -251,6 +251,90 @@ forms:
 `jcon fd <error_fd_to_jsonrpc>:`, `plugin-manager`;
 4. `log` is the context of the original log entry.
 
+#### `forward_event`
+
+A notification for topic `forward_event` is sent every time the status
+of a forward payment is set. The json format is same as the API
+`listforwards`.
+
+```json
+{
+  "forward_event": {
```

Comment:
Isn't adding a grouping object of the same name of the notification type a little bit redundant ? I would suggest to prefer
```
notification_name {
    "useful_field_a": "1",
    "useful_field_b": "2"
}
```
over
```
notification_name {
    notification_name {
        "useful_field_a": "1",
        "useful_field_b": "2"
    }
}
```

Reply:
Thank you, I'll remove it!

Reply:
Sorry, @darosior 
For the notification handle method, we need pass in a lot of parameters, If we remove the inner layer of object structure. This is inconvenient, especially there are some opptional  fields(like `resolved_time`).
For the above consideration, I think removing inner object structure may not be the best solution.
In order to avoid redundant , I‘d better change inner name, like "forward_payment".(I notice `invoice_payment hook` and `openchannel hook` also have the inner object structure).

Reply:
Yes and moreover as cdecker pointed out in #2823, adding a field in notifications would break plugins relying on it : we might be better off wrapping all notifications into objects.
About the name redundancy I think that if we keep the wrapping, we should keep the same field name as the notification name for all notifications.

Reply:
Sounds good! Reduce confusion.